analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/NullPointerDereference.CSharp6.cs

```diff
@@ -153,13 +153,13 @@ public void Method1(ReproForIssue2338 obj)
             switch (obj?.Color)
             {
                 case null:
-                    Console.ForegroundColor = obj.Color; // Noncompliant
+                    Console.ForegroundColor = obj.Color; // Non-compliant Suppressed #6117
                     break;
                 case ConsoleColor.Red:
-                    Console.ForegroundColor = obj.Color; // Noncompliant FP, requires nullability support in captured flow operation
+                    Console.ForegroundColor = obj.Color; // Non-compliant Suppressed #6117 FP, requires nullability support in captured flow operation
```

Comment:
Not `Noncompliant` and not `FP` now. The rest of the explanation still makes sense
```suggestion
                    Console.ForegroundColor = obj.Color; // Compliant Suppressed #6117 , requires nullability support in captured flow operation
```